scripts/exglobal_stage_ic.sh

```diff
@@ -106,7 +106,7 @@ for MEMDIR in "${MEMDIR_ARRAY[@]}"; do
 
     # Ocean Perturbation Files
     # Extra zero on MEMDIR ensure we have a number even if the string is empty
-    if (( 0${MEMDIR:3} > 0 )) && [[ "${OCN_ENS_PERTURB_FILES:-false}" == "true" ]]; then
+    if (( MEMBER > 0 )) && [[ "${STAGE_OCN_PERTURB_FILES:-false}" == "true" ]]; then
```

Comment:
```suggestion
    if (( 0${MEMDIR:3} > 0 )) && [[ "${STAGE_OCN_PERTURB_FILES:-false}" == "true" ]]; then
```
This was my mistake assuming `MEMBER` was defined earlier in this script.